src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,13 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import React from 'react';
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li key={comment.id}>
+        <CommentInfo comment={comment} />
+      </li>
```

Comment:
you can move this `li` inside the CommentInfo and set key directly to the CommentInfo